app/static/update-browser.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charSet="utf-8" />
+    <title>CleanBC Industrial Incentive Program</title>
+    <link rel="stylesheet" href="/static/bootstrap.min.css" />
+    <link
+      rel="apple-touch-icon"
+      href="/static/icons/bcid-apple-touch-icon.png"
+      sizes="180x180"
+    />
+    <link
+      rel="icon"
+      href="/static/icons/bcid-favicon-32x32.png"
+      sizes="32x32"
+      type="image/png"
+    />
+    <link
+      rel="icon"
+      href="/static/icons/bcid-favicon-16x16.png"
+      sizes="16x16"
+      type="image/png"
+    />
+    <link
+      rel="mask-icon"
+      href="/static/icons/bcid-apple-icon.svg"
+      color="#036"
+    />
+    <link rel="icon" href="/static/icons/bcid-favicon-32x32.png" />
+    <style>
+      header {
+        height: 66px;
+        background-color: #036;
+        color: #fff;
+        border-bottom: 2px solid #fcba19;
+      }
+      #logo {
+        height: 46px;
+        float: left;
+        margin: 10px 32px 10px 32px;
+      }
+      h1 {
+        margin-bottom: 32px;
+      }
+      h2 {
+        margin: 0;
+        line-height: 66px;
+      }
+      #heading {
+        background-color: #e9ecef;
+        padding: 64px 32px;
+      }
+      #browser-info {
+        padding: 64px;
+      }
+      .browser-logo {
+        width: 120px;
+      }
+      table {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <img id="logo" src="/static/BCID_CleanBC_rev_tagline_colour.svg" alt="logo for Province of British Columbia CleanBC"/>
+      <h2>CleanBC Industrial Incentive Program</h2>
+    </header>
+    <main>
+      <div id="heading">
+        <h1>Update your browser</h1>
+        <p>Secure use of site functionality requires an updated browser. Choose one of the links below to learn how to update your browser.</p>
+        <p>For mobile, please visit your device's app store.</p>
+      </div>
+      <div id="browser-info">
+        <table class="table">
```

Comment:
The table should have a description: https://www.w3.org/TR/WCAG20-TECHS/H39

Reply:
It isn't so much an actual data table, as it is an oldschool layout device (like in email templates) for bad browsers.

In response to browsers improvement over time, we can normally avoid table layout hacks that are bad for accessibility, but it remains an issue of course for a page that needs to be very backwards-compatible.

In this case, I think the "table" content (the browser links) should be accessibly described both by their alt text.

Reply:
It's interesting that Axe has never highlighted this as missing in all of our other tables 🤔

Reply:
This is one of the things that SonarCloud is complaining about.

Reply:
see https://sonarcloud.io/project/issues?id=bcgov_cas-ciip-portal&pullRequest=1163&resolved=false&types=BUG

Reply:
Ohhh, I hadn't noticed. Very strange that's taking this opportunity to complain nonetheless...

Reply:
Actually I guess SonarCloud doesn't render JSX and therefore only complains about this stuff for straight-up `.html` pages (of which there are zero others in this project).